qa/tasks/cephfs/test_cephfs_shell.py

```diff
@@ -38,13 +38,16 @@ def run_cephfs_shell_cmd(self, cmd, mount_x=None, shell_conf_path=None,
                              opts=None, stdin=None):
         if mount_x is None:
             mount_x = self.mount_a
-
         if isinstance(cmd, list):
             cmd = " ".join(cmd)
-
-        args = ["cephfs-shell"]
-        if shell_conf_path:
-            args += ["-c", shell_conf_path]
+        if not shell_conf_path:
+            conf_contents = "[cephfs-shell]\ncolors = False\ndebug = True\n"
+            confpath = mount_x.run_shell(args=['mktemp']).stdout.getvalue().\
+                strip()
+            sudo_write_file(mount_x.client_remote, confpath, conf_contents)
+            shell_conf_path = confpath
```

Comment:
Since we end up reusing this config multiple times, I'd suggest moving this code to `setUp` so it's run once per test.

Reply:
Nice suggestion. Made the change and tested it locally.